src/index.tsx

```diff
@@ -1494,7 +1494,7 @@ export class App extends React.Component<any, AppState> {
   }
 
   private saveDebounced = debounce(() => {
-    saveToLocalStorage(elements, this.state);
+    saveToLocalStorage(elements.filter(x => x.type !== "selection"), this.state);
```

Comment:
Looks like it doesn't pass lint. Could you run prettier by any chance?

Reply:
Weird that CSB didn't trigger the prettier.